frontend/lib/ui/style/table.css

```diff
@@ -59,6 +60,13 @@ table.overview_table {
   }
 }
 
+table.polling_stations_table {
+  td.number,
+  th.number {
+    width: 10rem;
+  }
+}
```

Comment:
On the polling station list on the polling station selector page the width of the number column is set to 6.5rem, which is long enough for 6 digit polling station numbers. Should we standardize this everywhere to the same width? The 6.5rem is set with an inline width, which should be set in css when the separate Table component is created I think.

Reply:
Expected max length for polling station numbers is 3 digits, so I would propose using the length of the 'Nummer' labels as the entity that dictates the width of this column. Anything wider is not needed; space should be used by the other columns. See screenshot. 
<img width="564" alt="Scherm­afbeelding 2024-10-18 om 11 01 42" src="https://github.com/user-attachments/assets/5ade252f-483a-4b5e-8626-55d5a11a42fe">

Reply:
Set to 6.5rem for now in https://github.com/kiesraad/abacus/pull/453/commits/26acb7b4e4300c1ec30d77b80e48b55669a0660d